operators/to_mapbox/__init__.py

```diff
@@ -0,0 +1,152 @@
+
+
+def unwind(from_key, to_key, key_transformer=None, resources=None):
+
+    # TODO - errors if to_key does not exist - using add_field below.
+    # TODO - from_key and to_key as equal length lists? like zip? See use case below with branch_name and organization_name.
+    # TODO - see if this is generic enough for the DF standard lib.
+    # TODO - always drop from_key, or, optionally drop?
+
+    def _unwinder(rows):
+        for row in rows:
+            try:
+                iter(row[from_key])
+                for value in row[from_key]:
+                    row[to_key] = value if key_transformer is None else key_transformer(value)
+                    yield row
+            except TypeError:
+                # no iterable to unwind, likely None. Take the value we have and set it on the to_key.
+                row[to_key] = row[from_key] if key_transformer is None else key_transformer(row[from_key])
+                yield row
+
+    def func(package):
+        matcher = ResourceMatcher(resources, package.pkg)
+        yield package.pkg
+        for r in package:
+            if matcher.match(r.res.name):
+                yield _unwinder(r)
+            else:
+                yield r
+
+    return func
+
+
+def mapbox_data_flow():
+    return DF.Flow(
+        load_from_airtable(settings.AIRTABLE_BASE, settings.AIRTABLE_RESPONSE_TABLE, settings.AIRTABLE_VIEW),
+        load_from_airtable(settings.AIRTABLE_BASE, settings.AIRTABLE_ORGANIZATION_TABLE, settings.AIRTABLE_VIEW),
+        load_from_airtable(settings.AIRTABLE_BASE, settings.AIRTABLE_LOCATION_TABLE, settings.AIRTABLE_VIEW),
+        load_from_airtable(settings.AIRTABLE_BASE, settings.AIRTABLE_BRANCH_TABLE, settings.AIRTABLE_VIEW),
+        load_from_airtable(settings.AIRTABLE_BASE, settings.AIRTABLE_SERVICE_TABLE, settings.AIRTABLE_VIEW),
+
+        filter_airtable_dummies(),
+        DF.update_package(**{'name': 'Mapbox Data'}),
+        DF.rename_fields({'__airtable_id': 'pk'}),
+
+        # RESPONSE PRE-PROCESSING
+        DF.update_resource(['Responses'], **{'name': 'responses', 'path': 'responses.csv'}),
+        DF.select_fields(['pk', 'id', 'name', 'description'], resources=['responses']),
+
+        # SERVICES PRE-PROCESSING
+        DF.update_resource(['Services'], **{'name': 'services', 'path': 'services.csv'}),
+        DF.filter_rows(lambda r: r['selected'] is True, resources=['services']),
+        DF.add_field('organization_pk', 'string', resources=['services']),
+        unwind('organizations', 'organization_pk', resources=['services']),
+        DF.add_field('response_pk', 'string', resources=['services']),
+        unwind('responses', 'response_pk', resources=['services']),
+        DF.select_fields(['pk','id', 'selected', 'name', 'organization_pk', 'response_pk'], resources=['services']),
+
+        # ORGANIZATION PRE-PROCESSING
+        DF.update_resource(['Organizations'], **{'name': 'organizations', 'path': 'organizations.csv'}),
```

Comment:
I think that `name='organizations', path='organizations.csv'` would be a little more readable here, no?

Reply:
agreed.